lib/src/widgets/article_list_screen/article_list_screen.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart' hide Card;
+import 'package:saraka/constants.dart';
+import 'package:saraka/widgets.dart';
+import 'package:provider/provider.dart';
+import 'package:saraka/blocs.dart';
+
+class ArticleListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) => Scaffold(
```

Comment:
Does `Scaffold` really need?

Reply:
I separated the task. I will consider it in Widget part.